webapp/components/Sidebar.js

```diff
@@ -59,21 +59,22 @@ class Sidebar extends PureComponent {
         // filter will first remove any plugins w/o sidebar property set to true
         // mapping through each parent item to create the sidebar nav element
         const sidebarItemProps = { ...plugin, theme, pathname };
-        if (plugin.subItems) {
-          // if this sidebar item has sub items
-          // then we need to create and append the children elements
-          sidebarItemProps.children = plugin.subItems.map(
-            (subItem, subIndex) => {
-              const props = {
-                ...subItem,
-                subItem: true,
-                pathName: subItem.pathName ? subItem.pathName : subItem.name,
-                theme,
-                key: `${index}-${subIndex}`
-              };
-              return this.renderNavItem(plugin, props);
-            }
-          );
+        if (plugin.parent) {
+          // const parentIndex = sidebarNavItems.findIndex(
```

Comment:
Was trying to support hierarchal paths, but it caused some other problems with other links since they are all relative paths. That will require some other changes so thought it would be better to include in a separate PR. This logic will still likely be needed though, so thought I'd just keep it for the next one.